packages/testing-helpers/test-web/fixture.test.js

```diff
@@ -63,6 +63,12 @@ describe('fixtureSync & fixture', () => {
     testElement(elementAsync);
   });
 
+  it('supports customized render functions', async () => {
+    const render = sinon.stub();
+    await fixture(html`<div></div>`, { render });
```

Comment:
We’d need a test of this using `scopedElements` as it gets pretty fancy internally.

Reply:
decided to not support this usecase, as discussed on slack